analysis_html_files/compound_w_gen_AndrewA.html

```diff
@@ -21,13 +21,7 @@ <h2>GENerating inputs</h2>
 <script>
 	// ---- Variables that need to be customized ----
 	
```

Comment:
Did you modify this file? or is it some kind of weird merge timeline issue?

Reply:
I did not modify this one.

Reply:
I checked the log and it seems I modified it on this branch 13 months back. So that's fine.